### ISSUE TITLE
grml-zsh-config: don't write notices about deltochar and mathfunc on minimalist systems (like OpenWRT)

### ISSUE THREAD

**Issue by Nowaker** [grml/grml#158]
```
Notice: no deltochar available :(
Notice: no mathfunc available :(
root@turris ~ #
```

It would be nice to have an option to suppress such warnings. There's nothing I can do about these missing modules as the OS I'm running grml-zsh-config on is OpenWRT.

**Comment by ft** [comment]
Agreed. I wonder whether we should silence these by default and add a way to enable them if a user do desires…

@mika WDYT?

In any case, I'd probably do it with some sort of a style. Not sure if I'll get to it during the week.

**Comment by mika** [comment]
@ft sounds good to me, thanks!

**Comment by Nowaker** [comment]
Awesome guys, thanks a ton!
